homeassistant/components/somfy/climate.py

```diff
@@ -126,7 +126,7 @@ def hvac_mode(self) -> str:
         """Return hvac operation ie. heat, cool mode."""
         if self._climate.get_regulation_state() == RegulationState.TIMETABLE:
             return HVAC_MODE_AUTO
-        return HVAC_MODES_MAPPING.get(self._climate.get_hvac_state())
+        return HVAC_MODES_MAPPING[self._climate.get_hvac_state()]
```

Comment:
This is a slight code change, @tetienne can you please double-check this change is OK?

Reply:
Yes it's OK, normally a value is always returned.

Reply:
Thanks for confirming!